lib/db.mjs

```diff
@@ -0,0 +1,27 @@
+import { MongoClient } from 'mongodb'
+
+let client, collection
```

Comment:
These variables are global to the file since both may need to be used in multiple functions. For now only `collection` is being used in this way, but `client` may need to be used in a future `closeConnection` function if needed.